test/three.source.unit.js

```diff
@@ -2,8 +2,6 @@
  * @author TristanVALCKE / https://github.com/Itee
  */
 
-import './unit/qunit-utils.js';
-
```

Comment:
I've realized that this is no necessary since the `UnitTests.html` already includes it.